lib/rules/not-html-matches.js

```diff
@@ -1,5 +1,5 @@
-function notHtmlMatches(node) {
-  return node.nodeName.toLowerCase() !== 'html';
+function notHtmlMatches(node, virtualNode) {
+  return virtualNode.props.nodeName.toLowerCase() !== 'html';
```

Comment:
`nodeName` is already normalized.

```suggestion
  return virtualNode.props.nodeName !== 'html';
```